signalfx-tracing/signalfx-python-tracing/django/README.md

```diff
@@ -0,0 +1,80 @@
+
+# Django Auto-Instrumentation Example
+
+The aim of this example is to show how to automatically produce distributed traces using the 
+[SignalFx Tracing Library for Python](https://github.com/signalfx/signalfx-python-tracing).
+For this example, using the [Django REST Framework](https://www.django-rest-framework.org/), 
+we have created a client/server interaction in the form of a basic blog service and consumer.
+
+
+
+## Installing and running the example app and client
+To run this example locally, and send traces to your available Smart Agent or Gateway, you may first clone this repository. 
+Next, navigate into the `django` folder and run the following commands:
+
+```bash
+$ pip install -r requirements.txt
+# install supported tracer and library instrumentation
+$ sfx-py-trace-bootstrap
+$ cd blog
+$ python manage.py makemigrations posts
+$ python manage.py migrate
+$ python manage.py createsuperuser --username <USERNAME> --email <EMAIL>
+$ python manage.py runserver localhost:<PORT>
+```
+To confirm everything is set up as expected, you may navigate to `http://localhost:<PORT>` 
+and sign in with the `username` and `password` you created above.
+
+
+### Interactions with the blog includes:
+    - Creating an article (user log in necessary)
+    - Viewing articles created
+    - Viewing list of contributing authors and their articles
+    - Updating/Editing an article  (user log in necessary)
+    - Deleting an article (user log in necessary)
+    
+
+### Trace identification:
+To clearly identify your traces, you may choose to name your server services by setting the
+ `SIGNALFX_SERVICE_NAME` environment variable to your preferred name. <br/>
+ 
+For example: 
+```bash
+$ export SIGNALFX_SERVICE_NAME='<YOUR_APP_NAME>'
+```
+The client's service name may be set in `create_tracer()` as follows:
+```python
+
+create_tracer(service_name='<APP_NAME>')
+```
+If not set, the default service name is: `SignalFx-Tracing`<br/><br/>
+ 
+### Running the example
+If necessary, navigate to the folder containing `manage.py` (in this example:`django/blog`) and run:
```

Comment:
When would this not be necessary?

Reply:
If the user is already in the folder 😀 . At this point in time, they probably are.... Re-worded it though, to remove any assumptions. 😁